client/assets/images/README.md

```diff
@@ -0,0 +1,8 @@
+# Images
+
+Images from this folder are managed by Webpack, and can be imported like this:
+
+```javascript
+import mediaPostImage from 'assets/images/upgrades/media-post.svg';
```

Comment:
I would call the variable `mediaPostImageURL` to make it clear that the value is a (relative) URL that can be used to create markup like `<img src={ mediaPostImageURL } />`.

Reply:
That's a very good idea 👍.